app/src/main/java/com/github/fribourgsdp/radio/data/view/SongFragment.kt

```diff
@@ -41,14 +41,14 @@ open class SongFragment : MyFragment(R.layout.fragment_song) {
         return MusixmatchLyricsGetter
     }
 
-    private fun fetchLyrics(lyricsGetter: LyricsGetter) {
+    /*private fun fetchLyrics(lyricsGetter: LyricsGetter) {
         lyricsGetter.getLyrics(song.name, song.artist)
             .thenAccept{f ->
                 currentLyrics = f
                 doSaveLyrics = true
                 updateLyrics(requireView().findViewById(R.id.editTextLyrics))
             }
-    }
+    }*/
```

Comment:
Can be removed if no longer useful

Reply:
Yes I will remove it.

Reply:
As discussed over any channel of discussion, this functionality should actually stay so I will reput it.